apps/transformers/lib/transformations/date_time.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Transformers.DateTime do
+  @behaviour Transformation
+
+  alias Transformations.FieldFetcher
+
+  @impl Transformation
+
+  def transform(payload, parameters) do
+    with {:ok, sourceField} <- FieldFetcher.fetch_parameter(parameters, :sourceField),
+         {:ok, sourceFormat} <- FieldFetcher.fetch_parameter(parameters, :sourceFormat),
+         {:ok, targetField} <- FieldFetcher.fetch_parameter(parameters, :targetField),
+         {:ok, targetFormat} <- FieldFetcher.fetch_parameter(parameters, :targetFormat),
+         {:ok, payloadSourceValue} <- FieldFetcher.fetch_value(payload, sourceField),
+         {:ok, sourceDatetime} <- parseTime(payloadSourceValue, sourceFormat, sourceField),
+         {:ok, transformedDatetime} <- formatDateTime(sourceDatetime, targetFormat) do
+      {:ok, payload |> Map.put(targetField, transformedDatetime)}
+    else
+      {:error, reason} ->
+        {:error, reason}
+
+      nil_payload ->
+        {:ok, nil_payload}
+    end
+  end
+
+  defp parseTime(dateString, dateFormat, sourceField) do
+    with {:ok, result} <- Timex.parse(dateString, dateFormat) do
+      {:ok, result}
+    else
+      {:error, timexReason} ->
+        {:error,
+         "Unable to parse datetime from \"#{sourceField}\" in format \"#{dateFormat}\": #{
+           timexReason
+         }"}
+    end
+  end
+
+  defp formatDateTime(dateTime, format) do
```

Comment:
super tiny thing - I think that elixir convention for functions is snake_case?

Reply:
glad you mentioned this, my JS is showing. Fixed!

Reply:
Hah same, I find myself doing this all the time too!